Src/Scripting/NativeDefinitions/MathDefinitions.cpp

```diff
@@ -157,6 +163,8 @@ MathDefinitions::MathDefinitions(ScriptManager* mgr) {
     engine->RegisterObjectMethod("Vector3f", "Vector3f opMul(float f) const", asMETHODPR(PGE::Vector3f, operator*, (float) const, PGE::Vector3f), asCALL_THISCALL);
     engine->RegisterObjectMethod("Vector3f", "Vector3f opMul_r(float f) const", asMETHODPR(PGE::Vector3f, operator*, (float) const, PGE::Vector3f), asCALL_THISCALL);
     engine->RegisterObjectMethod("Vector3f", "Vector3f opDiv(float f) const", asMETHODPR(PGE::Vector3f, operator/, (float) const, PGE::Vector3f), asCALL_THISCALL);
+    engine->RegisterObjectMethod("Vector3f", "Vector3f opMul(const Vector3f&in other) const", asMETHODPR(PGE::Vector3f, operator*, (float) const, PGE::Vector3f), asCALL_THISCALL);
+    engine->RegisterObjectMethod("Vector3f", "Vector3f opDiv(const Vector3f&in other) const", asMETHODPR(PGE::Vector3f, operator/, (float) const, PGE::Vector3f), asCALL_THISCALL);
```

Comment:
These overloads are not permissible.

Reply:
This has been reverted for the time being.